ice40/utils/ice40_generate_routing.py

```diff
@@ -380,20 +373,13 @@ def __str__(self):
 
     #span4_horz_{0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47}
     io_tracks.extend([
```

Comment:
need to remove this line too. Although really this file isn't used and we should probably do some more purging.

Reply:
Both `span4_horz_` and `span12_horz_`?
https://github.com/SymbiFlow/symbiflow-arch-defs/blob/6b226e96da87bb2faa112fda25ef47847ecf3825/ice40/utils/ice40_generate_routing.py#L374-L387

Reply:
I think just line 375. You just missed the open line of the block you removed, so `check_python` is failing. You should probably run `make format_python` and check in as well.

Reply:
Oh right, I missed that, thanks